dags/utils/slack_operator.py

```diff
@@ -1,11 +1,18 @@
 from airflow.hooks.base_hook import BaseHook
 from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
+from pendulum import timezone
 
 # This is the Conn Id that we set when creating the connection in the Airflow dashboard
 # in Admin > Connections.
 SLACK_CONN_ID = "slack"
 
 
+def get_central_time_exec_data(context):
```

Comment:
is `context` an object that airflow passes around to every function? I keep seeing this and don't fully understand

Reply:
i guess the `context` is passed as the only argument when a hook executes.

it's a little different for tasks, where the entire context gets passed as keyword arguments after your custom arguments, so you can do [this](https://github.com/cityofaustin/atd-airflow/blob/4c2d4b6d067e1e77053480214c7856e13b1979d5/dags/utils/knack.py#L8). it's annoyingly hard to find this in the documentation :/

Reply:
oh yeah, good call - here are some docs on it https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/taskflow.html#context

Reply:
ah that makes sense ty John 🧙, is the only reason you had to `**kwarg` it there was because of the `should_replace_monthly` arg or because it's a task?

Reply:
thanks @mddilley!

Reply:
you are right that the docs are not specific enough about using context, John! thanks for adding the examples, and, to me, the docs read like your last example should work too.